examples/k8s-bookinfo.yaml

```diff
@@ -56,8 +56,10 @@ spec:
           value: http://$(CONTROLLER_SERVICE_HOST):$(CONTROLLER_SERVICE_PORT)
         - name: A8_DISCOVERY_BACKEND
           value: kubernetes
-        - name: A8_SERVICE
-          value: gateway:none
+        - name: A8_KUBERNETES_POD_NAME
```

Comment:
So, what is the effective content of the "tags" field? The sidecar launches nginx with A8_SERVICE="<name>:<tags>"..
If the "gateway:none" changes into something else, then please update docker-bookinfo/helloworld.yaml and the *.json files in examples folder.